spaceship/lib/spaceship/client.rb

```diff
@@ -608,6 +608,8 @@ def detect_most_common_errors_and_raise_exceptions(body)
       # Check if the failure is due to missing permissions (iTunes Connect)
       if body["messages"] && body["messages"]["error"].include?("Forbidden")
         raise_insuffient_permission_error!
+      elsif body["messages"] && body["messages"]["error"].include?("insufficient privileges")
+        raise_insuffient_permission_error!(caller_location: 3)
```

Comment:
I think specifying `caller_location` is ok here, but lets add some comments around why you're passing 3, and what the effect is.